static/src/stylesheets/module/content/_content.global.scss

```diff
@@ -182,6 +182,14 @@
             float: left;
         }
     }
+
+    //Todo: Replace to content--gallery when gallery redesign goes live
+    .new-gallery & {
+        position: relative;
+        left: 0;
+        border-top: 0;
+        padding: $gs-baseline 0 0;
+    }
```

Comment:
All other styles under `new-gallery` are currently in `gallery.scss`. Should these go there for consistency?